src/auto-evo/EvolutionaryTree.cs

```diff
@@ -156,14 +163,21 @@ public override void _Ready()
         latoSmallRegular = (Font)GD.Load("res://src/gui_common/fonts/Lato-Regular-Small.tres");
     }
 
-    public void Init(Species luca, string? updatedLUCAName = null)
+    public void Init(IEnumerable<Species> initialSpecies, uint playerSpeciesId = 1,
+        string? updatedPlayerSpeciesName = null)
     {
-        SetupTreeNode(luca, null, 0);
+        foreach (var species in initialSpecies)
```

Comment:
What if initial species is empty? Does this tree work correctly or should this throw an exception?

Reply:
The tree will not work correctly, but I can't figure out an elegant way to check if the IEnumerable is empty.

Reply:
I recommend adding a bool called "seenASpecies = false" that gets set to true inside the loop, then after the loop throw if the variable is false.